alert-common/src/main/java/com/synopsys/integration/alert/common/rest/model/FieldModel.java

```diff
@@ -30,6 +30,7 @@
 
 public class FieldModel extends Config {
     private Map<String, FieldValueModel> keyToValues;
+    // TODO DescriptorKey and a ConfigContextEnum should be available instead of Strings
```

Comment:
Some `FieldModel`s are built using the `ConfigContextEnum` and `DescriptorKey` classes, but other FieldModel instantiations are from String sources. I can go through and attempt to convert most `FieldModel` instantiations to use `ConfigContextEnum` and `DescriptorKey` classes, but would need some help in the tricky spots.